pkg/apis/operator/v1alpha1/tektonresult_types.go

```diff
@@ -58,8 +58,6 @@ type TektonResultSpec struct {
 // ResultsAPIProperties defines the fields which are configurable for
 // Results API server config
 type ResultsAPIProperties struct {
-	DBUser                string `json:"db_user,omitempty"`
-	DBPassword            string `json:"db_password,omitempty"`
```

Comment:
@khrm `db_user` and `db_password` not included in #1547 
Added support only to `S3_*`

Can you please check that?

Reply:
https://github.com/tektoncd/operator/pull/1547/files#diff-75f9ef7e65d67d81f17ba1a762c815f827b0d082c48d7e35326f9c4579f40afbR50-R58

Reply:
`db_user` and `db_password` comes from `tekton-results-postgres` .  Remember, earlier Siamak or Koustav had mentioned handling both s3 as well as db. But we didn't need to do anything about DB as it is already taken from secret.

Reply:
We are more strict on those. Results installation won't proceed if this secret isn't present.

Reply:
from which secret those values come from?